src/lmrtfy.js

```diff
@@ -9,6 +9,15 @@ class LMRTFY {
         default: true,
         onChange: (value) => LMRTFY.onThemeChange(value)
       });
+      
+      Handlebars.registerHelper('controlledToken', function (actor) {
```

Comment:
It would be best to namespace this manually with `lmrtfy-controlledToken` to avoid potential conflicts.

Reply:
ah I see, good call, just pushed this change.